components/brave_wallet_ui/components/desktop/popup-modals/sell-asset-modal/sell-asset-modal.tsx

```diff
@@ -107,9 +109,23 @@ export const SellAssetModal = (props: Props) => {
     return Number(sellAmount) > new Amount(fiatBalance.format(2)).toNumber()
   }, [sellAmount, fiatBalance])
 
+  const meetsMinimumSellThreshold = React.useMemo(() => {
+    return Number(sellAmount) >= MINIMUM_SELL_THRESHOLD
+  }, [sellAmount])
+
   const isSellButtonDisabled = React.useMemo(() => {
-    return sellAmount === '' || insufficientBalance || hasTooManyDecimals
-  }, [sellAmount, insufficientBalance, hasTooManyDecimals])
+    return sellAmount === '' || insufficientBalance || hasTooManyDecimals || !meetsMinimumSellThreshold
+  }, [sellAmount, insufficientBalance, hasTooManyDecimals, meetsMinimumSellThreshold])
+
+  const errorMessage = React.useMemo(() => {
+    if (!meetsMinimumSellThreshold && sellAmount !== '') {
+      return getLocale('braveWalletSellMinimumAmount').replace('$1', new Amount(MINIMUM_SELL_THRESHOLD).formatAsFiat(defaultCurrencies.fiat))
+    }
+    if (insufficientBalance) {
+      return getLocale('braveWalletNotEnoughBalance').replace('$1', selectedAsset.symbol)
+    }
+    return ''
+  },[selectedAsset.symbol, meetsMinimumSellThreshold, sellAmount, defaultCurrencies.fiat, insufficientBalance])
```

Comment:
should this go in the `useMultiChainSellAssets` hook?

Reply:
Kept in `sell-asset-modal.tsx` to reduce complexity.